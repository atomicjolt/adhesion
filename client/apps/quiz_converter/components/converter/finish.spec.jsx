import React from 'react';
import { shallow } from 'enzyme';
import Finish from './finish';

describe('Finish page', () => {
  let result;

  beforeEach(() => {
    const props = {};
    result = shallow(<Finish {...props} />);
  });

  it('matches the snapshot', () => {
    expect(result).toMatchSnapshot();
  });
});
