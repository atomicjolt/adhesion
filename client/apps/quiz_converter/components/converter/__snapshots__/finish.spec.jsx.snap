// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Finish page matches the snapshot 1`] = `
ShallowWrapper {
  "length": 1,
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <Finish />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": <h1
        className="c-title"
>
        Redirecting to Quiz...
</h1>,
      "className": "c-contain finish",
    },
    "ref": null,
    "rendered": Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "Redirecting to Quiz...",
        "className": "c-title",
      },
      "ref": null,
      "rendered": "Redirecting to Quiz...",
      "type": "h1",
    },
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": <h1
          className="c-title"
>
          Redirecting to Quiz...
</h1>,
        "className": "c-contain finish",
      },
      "ref": null,
      "rendered": Object {
        "instance": null,
        "key": undefined,
        "nodeType": "host",
        "props": Object {
          "children": "Redirecting to Quiz...",
          "className": "c-title",
        },
        "ref": null,
        "rendered": "Redirecting to Quiz...",
        "type": "h1",
      },
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactFifteenAdapter {
      "options": Object {
        "supportPrevContextArgumentOfComponentDidUpdate": true,
      },
    },
  },
}
`;
